Components/NavigationFooter/_style.scss

```diff
@@ -1,6 +1,24 @@
 [is='flynt-navigation-footer'] {
-  .navigation {
-    padding: $component-spacing-mobile 0;
+  font-size: 0.75rem;
+
+  .container {
+    padding-bottom: $component-spacing-mobile;
+    padding-top: $component-spacing-mobile;
+    text-align: center;
+
+    @media (min-width: $breakpoint-tablet) {
+      display: flex;
+      flex-direction: row-reverse;
+      justify-content: space-between;
```

Comment:
If there are no nav items, the copyright notice will move to the right. Minor to me, as there "should" be nav items, but just saying that this might be unintentional.

Reply:
Yeah I didn't want to add more complexity to it when this anyway *shouldn't* happen.